src/IPHlpApi/IPHlpApi+TcpTableType.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="TcpTableType"/> nested type.
+    /// </content>
+    public static partial class IPHlpApi
+    {
+        /// <summary>
+        /// The <see cref="TcpTableType"/> enumeration defines the set of values used to indicate the type
+        /// of table returned by calls to <see cref="IPHlpApi.GetExtendedTcpTable"/>.
+        /// </summary>
+        /// <seealso href="http://msdn2.microsoft.com/en-us/library/aa366386.aspx"/>
+        public enum TcpTableType
```

Comment:
This enum is one of the few that actually *is* defined in native code, so we should use the name it defines (`TCP_TABLE_CLASS`).